<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joinwindow</class>
 <widget class="QDialog" name="joinwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="pwEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>85</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#006636;
font-weight:700;</string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>115</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/logo1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>155</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
color:#fff;
font-weight:600;
background-color:#2c2a29;
cursor:pointer;
}
QPushButton:hover{
background-color:#4c4a49;
}

</string>
   </property>
   <property name="text">
    <string>중복확인</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>811</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#f6f5ef;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2c2a29;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repwEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>265</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>PW 재입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>325</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
color:#fff;
font-weight:600;
background-color:#006636;
cursor:pointer;
}
QPushButton:hover{
background-color:#108656;
}</string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coffeeButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>15</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
font-weight:400;
}
QPushButton:hover{

}</string>
   </property>
   <property name="text">
    <string>COFFEE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="orderButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>15</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
font-weight:400;
}
QPushButton:hover{

}</string>
   </property>
   <property name="text">
    <string>TEA</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #ddd;</string>
   </property>
  </widget>
  <zorder>textBrowser_4</zorder>
  <zorder>joinButton</zorder>
  <zorder>textBrowser_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>idEdit</zorder>
  <zorder>pwEdit</zorder>
  <zorder>coffeeButton</zorder>
  <zorder>repwEdit</zorder>
  <zorder>orderButton</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>checkButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
